source/reference/template-apps.txt

```diff
@@ -78,34 +90,117 @@ option is most convenient for you.
             }
          }
 
-.. _explore-template-app:
-
-Explore a Template App
-----------------------
+.. _get-template-app-client:
 
-When you create a template app, the UI includes a
-:guilabel:`Welcome to Your Application` section to help you get started
-with services and front-end code. You can:
+Get a Template App Client
+-------------------------
 
-1. Learn more about what your chosen template app includes.
+Some of the template apps come with working clients to explore and build 
+on when creating your applications. You can access these clients in a few
+ways:
 
-2. Follow on-screen guides to customize your app.
+.. tabs::
+   
+   .. tab:: Atlas App Services UI
+      :tabid: ui
+      
+      **Download the client when you create the template app**
+
+      When you create a template app using the App Services UI, the UI 
+      prompts you with two options to get the client code immediately 
+      after creating the template:
+
+      - Download the client code in your preferred programming language 
+        as a .zip file.
+      - Use the {+cli+} to pull the client code in your preferred programming 
+        language.
+
+      Follow the on-screen instructions to get the client code using one 
+      of these methods. 
+
+      When you use this method to get the client code, the client code is not 
+      pre-populated with your App Services App ID. To use one of these
+      clients, you must :ref:`find your App ID <find-your-app-id>` to use 
+      in the client. Follow the :guilabel:`Configuration` and 
+      :guilabel:`Download the Client as a Zip File` instructions in the client 
+      :file:`README.md` to learn where to insert your App ID.
```

Comment:
[non-blocking, but substantial] I stumbled when I got to this sentence. My first thought was that this was meant for one of the list items above. Then I thought "this method" meant using the App Services UI. Then, I wondered if it was referring to "Download the client when you create the template app".

It _seems_ to be associated with the tab-level context. As such, I think we could improve the flow (and reduce duplication) by moving this info to the top of the UI and Realm CLI tabs. So, for Realm CLI, above "Download the client when you create the template app" we could say something like this:
> When you download client code through the App Services UI, the client code is pre-populated with your App Services App ID.

For the App Services UI tab, we could move this whole paragraph up to the top of the tab.

Reply:
Makes sense!